src/main/java/com/ldtteam/structurize/api/ItemStackUtils.java

```diff
@@ -81,24 +82,7 @@ public static List<ItemStack> getItemStacksOfTileEntity(final CompoundTag compou
             }
         }
 
-        return items;
-    }
-
-    @NotNull
-    private static List<ItemStack> getItemStacksFromNbt(@NotNull final CompoundTag compound, final HolderLookup.Provider provider)
-    {
-        final List<ItemStack> items = new ArrayList<>();
-        final ListTag listtag = compound.getList("Items", Tag.TAG_COMPOUND);
-
-        for (int i = 0; i < listtag.size(); ++i)
-        {
-            final CompoundTag compoundtag = listtag.getCompound(i);
-            final ItemStack stack = ItemStack.parseOptional(provider, compoundtag);
-            if (!stack.isEmpty())
-            {
-                items.add(stack);
-            }
-        }
+        SingleBlockFakeLevelGetter.unset(itemHandlerFakeLevel, tileEntity);
```

Comment:
prepare + unset seems kinda unstable, isnt there any single shot message we could use?